src/utilities/util_verify.c

```diff
@@ -12,11 +12,14 @@ static int
 usage(void)
 {
     static const char *options[] = {"-d config",
-      "display underlying information during verification", "-s",
-      "verify against the specified timestamp", NULL, NULL};
+      "display underlying information during verification", "-c",
+      "continue to the next page after encountering error during verification", "-s",
+      "verify against the specified timestamp", "-t", "do not clear txn ids during verification",
```

Comment:
Why are we adding the text related to `-t` option here? I don't think this functionality exists in the verify utility for version 5.0.

Reply:
This is a mistake, sorry. Thank you for finding this.

Reply:
I have removed it from the backport, should be gone now.